### ISSUE TITLE
Pod management tabs disappearing after 2.8 update

### ISSUE THREAD

**Issue by HinomaruIL** [pods-framework/pods#6266]
As discussed on Slack, after updating from 2.7.3 to 2.8.3 using the wordpress plugin updater, all tabs for managing pods except for fields and REST API disappeared. Issue persisted after rolling back or trying to install a different version of the plugin.
In my case restoring from backup and updating again did not replicate the issue. 

```json
`{
    "meta": {
        "version": "2.8.0-rc-3",
        "build": 1635615598
    },
    "pods": [
        {
            "name": "animot",
            "id": 59414,
            "label": "\u05d0\u05e0\u05d9\u05de\u05d4",
            "description": "",
            "type": "post_type",
            "storage": "meta",
            "label_singular": "\u05d0\u05e0\u05d9\u05de\u05d4",
            "public": "1",
            "show_ui": "1",
            "rest_enable": "1",
            "supports_title": "1",
            "supports_editor": "1",
            "object": "animot",
            "_migrated_28": "1",
            "read_all": "1",
            "groups": [
                {
                    "name": "d7a9d793d795d7aa_d7a0d795d7a1d7a4d799d79d",
                    "id": 59415,
                    "label": "\u05e9\u05d3\u05d5\u05ea \u05e0\u05d5\u05e1\u05e4\u05d9\u05dd",
                    "description": "",
                    "weight": 0,
                    "object_storage_type": "post_type",
                    "fields": []
                }
            ]
        }
    ]
}`
```

**Comment by JoryHogeveen** [comment]
Hi @HinomaruIL 

> In my case restoring from backup and updating again did not replicate the issue.

Do you mean you cannot reproduce this issue yourself?

**Comment by HinomaruIL** [comment]
> Hi @HinomaruIL
> 
> > In my case restoring from backup and updating again did not replicate the issue.
> 
> Do you mean you cannot reproduce this issue yourself?

Hi @JoryHogeveen , that's correct, but @sc0ttkclark asked me to report it anyway to be fixed 'so that the detection of extended objects doesn't rely on that field and used init detection going forward'.

**Comment by JoryHogeveen** [comment]
@HinomaruIL 
Check, thanks for letting me know!

**Comment by sc0ttkclark** [comment]
Fixed via 950f56fe2639fecc53e2924076bc9350cf52261c in which it better handles dynamically whether a content type is extended or not.
